packages/design/src/Colors.mdx

```diff
@@ -24,4 +24,135 @@ ensure you are using colors for their intended purpose.
   alongside color to ensure your content can be understood by everyone.
 </Banner>
 
+## Usage Guidelines
+
+### Typography
+
+Typographic elements in Jobber should use consistent colors to ensure
+readability.
+
+#### Heading
+
+Headings have a bold, dark color to cement their hierarchy.
+
+#### Text
+
+A slightly softer color is used for body text for greater readability.
+
+##### Text--Supporting
+
+Text that is relevant but less important can be lower-contrast to suggest its’
+reduced importance.
+
+### Interactive
+
+Use these colors in buttons and form controls to communicate the presence and
+meaning of interaction.
+
+#### Interactive
+
+The default color used for interactive elements.
+
+#### Destructive
+
+Use to signfify that an interaction will destroy something in the users’ account
+or workflow.
+
+#### Cancel
+
+Use to signify that an interaction will cancel or exit a workflow
+
+#### Disabled
+
+Use to signify that an interactive element is disabled.
+
+##### Disabled--Secondary
+
+Use when a disabled element needs more than one shade.
+
+#### Focus
+
+Use to indicate that an element has been focused.
+
+### Status
+
+Use these colors in labels, icons, filters, alerts, and other elements where
+color can add meaning to the state of the system or an item in the system.
+
+All status colors have a main color, a surface color, and an on-surface color.
+The on-surface color should be used for an element when it sits inside of an
+element with the status' surface color to maintain a cohesive tone and ensure
+sufficient color contrast.
+
+#### Critical
+
+Action required; user must see this status to be unblocked.
+
+#### Warning
+
+Action _may_ be required as a consequence of current state.
+
+#### Success
+
+No action required; an action has completed successfully.
+
+#### Informative
+
+No action required; but helpful to know about.
+
+#### Inactive
+
+No action required; not part of an active workflow.
+
+### Surfaces
+
+Surfaces are the background-colors of almost every element in Jobber.
+
+#### Surface
+
+Most elements in Jobber have a white surface; if clickable, they have a light
```

Comment:
again here, not sure if we want to be color specific?

Reply:
👍

Reply:
Changed this to "light" b/c even in dark mode, the foreground surfaces, will be lighter than backgrounds.